build_info/rust_config.toml

```diff
@@ -7,7 +10,7 @@ targets = "AArch64;ARM;X86"
 host = ["PROCURSUS_TARGET"]
 build-dir = "PROCURSUS_BUILD_DIR"
 extended = true
-tools = ["cargo", "clippy", "analysis", "src"]
```

Comment:
rust-analysis is no longer included because rls is not really the recommended language server anymore